apps/address-service/domains/address/schema/ActualizeAddressesService.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Generated by `createservice address.ActualizeAddressesService --type mutations 'addresses:[AddressWhereUniqueInput!]!'`
+ */
+
+const get = require('lodash/get')
+
+const { getLogger } = require('@open-condo/keystone/logging')
+const { GQLCustomSchema, getById, find, getByCondition } = require('@open-condo/keystone/schema')
+
+const access = require('@address-service/domains/address/access/ActualizeAddressesService')
+const { Address, AddressSource } = require('@address-service/domains/address/utils/serverSchema')
+const { DADATA_PROVIDER, GOOGLE_PROVIDER } = require('@address-service/domains/common/constants/providers')
+const { generateAddressKey } = require('@address-service/domains/common/utils/addressKeyUtils')
+const { DadataSuggestionProvider } = require('@address-service/domains/common/utils/services/suggest/providers')
+
+const logger = getLogger('ActualizeAddressesService')
+
+const ActualizeAddressesService = new GQLCustomSchema('ActualizeAddressesService', {
```

Comment:
It's just a question - why not update?) IMO "Actualize" sounds a bit strange

Reply:
The update sounds like we should pass some data on to update the model. But here we actualize the model with data loaded in runtime.